operator/Makefile

```diff
@@ -78,9 +78,18 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 
 generate: controller-gen kustomize fmt ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hacks/boilerplate.go.txt" paths="./..."
+
+    # Generating installer manifest file : ibm-spectrum-scale-csi/generated/installer/ibm-spectrum-scale-csi-operator.yaml
 	$(KUSTOMIZE) build --reorder none config/default > ../generated/installer/ibm-spectrum-scale-csi-operator.yaml
 	$(KUSTOMIZE) cfg fmt ../generated/installer/ibm-spectrum-scale-csi-operator.yaml
 	sed -i "s|0.0.0|$(VERSION)|g" ../generated/installer/ibm-spectrum-scale-csi-operator.yaml
+
+    # Generating installer manifest file : ibm-spectrum-scale-csi/generated/installer/ibm-spectrum-scale-csi-operator-excluding-scc.yaml
+	$(KUSTOMIZE) build --reorder none config/operator/delete_operator_security_context_constraints > ../generated/installer/ibm-spectrum-scale-csi-operator-excluding-scc.yaml
```

Comment:
should we say it ibm-spectrum-scale-csi-operator-k8s.yaml ? and are we generating dev.yaml for without scc also ?

Reply:
Changed manifest name to `ibm-spectrum-scale-csi-operator-k8s.yaml`. For dev added manifest `ibm-spectrum-scale-csi-operator-k8s-dev.yaml`